### PR TITLE
Adding contributing info

### PR DESCRIPTION
**Pull request by markemeis**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Contributions are welcome and much appreciated. Just follow these easy steps to contribute.
+
+## Code Standard and Guideline
+For consistency, we ask that you adhere to some basic code guidelines when contributing to the Kubeiql. Please run [gofmt](https://golang.org/cmd/gofmt/) on your code before submitting a pull request.
+
+## Our Development Process
+
+### Pull Requests
+We welcome and encourage pull requests. When we get a pull request, it is peer reviewed and sent to QA for integration and smoke testing. Next, a core team member sign offs and performs the pull request merge. We'll provide updates and feedback throughout the process to keep you informed.
+
```
Comment (jd3quist) [inline CONTRIBUTING.md]: Should be "signs off" instead of "sign offs"...?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Before filing a new issue, check Known Issues to see if your problem already exists. When reporting a new issue, provide as much detail as possible. The more information, the easier it is to debug and the faster you'll get a fix. 
+
+**Tips:**
+
+* A description. What did you expect to happen? What actually happened? Why do you think the behavior was incorrect?
+* What Kubernetes version you are running and any other versions you have seen the issue in
+* Anything else that seems relevant. 
+
+## License
+By contributing to iOS Mobile SDK, you agree that your contributions will be licensed under its [license][license].
```
Comment (jd3quist) [inline CONTRIBUTING.md]: Not "iOS Mobile SDK" surely...
Also, confirm that the license link is what we want....?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+[cla]: https://www.clahub.com/agreements/yipeeio/kubeiql
+[license]: /LICENSE
+
```
Comment (jd3quist) [inline CONTRIBUTING.md]: Does this refer to a repo file named "LICENSE"?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+### Known Issues
+We also use GitHub issues for updates to known issues, including alerts when fixes are in progress. 
+
+### Reporting New Issues
+Before filing a new issue, check Known Issues to see if your problem already exists. When reporting a new issue, provide as much detail as possible. The more information, the easier it is to debug and the faster you'll get a fix. 
+
+**Tips:**
+
+* A description. What did you expect to happen? What actually happened? Why do you think the behavior was incorrect?
+* What Kubernetes version you are running and any other versions you have seen the issue in
```
Comment (jd3quist) [inline CONTRIBUTING.md]: How about this instead.  Seems like better sentence structure and more consistent with prior tone...

What Kubernetes version are you running?  Have you seen the same issue/behavior on other versions too?

File: CONTRIBUTING.md (hunk 3 above)
Comment (markemeis) [reply CONTRIBUTING.md]: That is just a markdown link

File: CONTRIBUTING.md (hunk 4 above)
Comment (markemeis) [reply CONTRIBUTING.md]: Yes.  That's better.

**Pull request merged**